doc/nrf/known_issues.rst

```diff
@@ -2456,7 +2455,7 @@ tx_buffer_length set incorrectly
 Trusted Firmware-M (TF-M)
 *************************
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
In NCS 2.2.0 the 0.14.2 toolchain is not allowed, and issue is fixed for 0.15.0.

This should be marked as fixed in 2.2.0

Reply:
Edited.